packages/extension-api/src/extension-api.d.ts

```diff
@@ -1953,13 +1961,9 @@ declare module '@podman-desktop/api' {
     export function stopContainer(engineId: string, id: string): Promise<void>;
     export function deleteContainer(engineId: string, id: string): Promise<void>;
     export function buildImage(
-      containerBuildContextDirectory: string,
-      relativeContainerfilePath: string,
-      imageName: string,
-      platform: string,
-      selectedProvider: ProviderContainerConnectionInfo | containerDesktopAPI.ContainerProviderConnection,
+      context: string,
```

Comment:
AFAIK why rename `containerBuildContextDirectory` to `context` ?

Reply:
We are calling the method `buildImage`, that seems redundant to say again build, and container, and the context is always a directory when building an image.

In the same way:
- `selectedProvider` is renamed to `provider`, since we are providing it, it seems logic it is the one we want to _select_
- `imageName` is renamed to `target`, to match `podman | docker build` arguments
- `relativeContainerfilePath` is renamed to `containerfile` because using `file` (--file) is not really explicit enough

Reply:
i've just notice that `--target` is not the equivalent `-t`. The shortcut `-t` is for tag. I will then rename the target property to `tag`